src/Statistics/BinomialConfidenceInterval.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "Statistics/BinomialConfidenceInterval.h"
+
+#include <cmath>
+#include <cstdint>
+
+namespace orbit_statistics {
+
+double sqr(double x) { return x * x; }
+
+[[nodiscard]] BinomialConfidenceInterval WilsonBinomialConfidenceIntervalEstimator::Estimate(
+    float ratio, uint32_t trials) const {
```

Comment:
Could it be useful for `trials` to be `uint64_t`?

Reply:
No, I am afraid. As `CallstackData::GetCallstackEventsCount` returns `uint32_t `. And it doesn't feel like a big issue. At 10 kHz sampling rate `uint32_t` should be enough for 119 hours of capture.

Reply:
I meant that it could be used for something other than callstacks samples (but if you think that `uint32_t` is enough in all cases then there's nothing to do).

Reply:
If `trials == 2**32` the width of the confidence interval is 3e-5 or less. So I guess, it's enough.